product_docs/docs/pgd/5/node_management/maintainance_with_proxies.mdx

```diff
@@ -4,57 +4,57 @@ title: Maintenance commands through proxies
-Therefore it is best to run maintenance commands on any node in a group that isn't the write leader.
-Generally, this requires you connect directly and issue the maintenance commands on the non-write leader nodes.
-But there are situations where this is not possible.
+Therefore, it's best to run maintenance commands on any node in a group that isn't the write leader.
+Generally, this requires you to connect directly and issue the maintenance commands on the non-write-leader nodes.
+But in some situations, this isn't possible.
 
 ## Maintenance and proxies
 
 Proxies, by design, always connect to and send commands to the current write leader.
-This would usually mean that you should not connect via a proxy to perform maintenance.
-PGD clusters nodes are able to present a direct connection for psql and pgd cli clients which can be used for issuing maintenance commands to the server on those nodes. 
+This usually means that you must not connect to a proxy to perform maintenance.
+PGD clusters nodes can present a direct connection for psql and PGD CLI clients that you can use to issue maintenance commands to the server on those nodes. 
 But there are environment in which the PGD cluster is deployed where a proxy is the only way to access the cluster.
 
 For example, in BigAnimal, PGD clusters are locked down such that the only access to the database is through an instance of PGD Proxy. 
-This reduces the footprint of the cluster and makes it more secure but it does require a different way of sending maintenance requests to the cluster’s nodes.
+This configuration reduces the footprint of the cluster and makes it more secure. However, it requires that you use a different way of sending maintenance requests to the cluster's nodes.
 
-The technique outlined here is generally useful for despatching commands to specific nodes without being directly connected to that node’s server.
+The technique outlined here is generally useful for despatching commands to specific nodes without being directly connected to that node's server.
 
 ## Maintenance commands
 
-When we refer to maintenance commands, we are referring to:
+The term *maintenance commands* refers to:
 
 * `VACUUM` 
-* Non-replicated DDL commands (which you may want to manually replicate)
+* Non-replicated DDL commands (which you might want to manually replicate)
 
 
 ## A note on node names
 
-We will be addressing the servers in the cluster by their PGD cluster node names. To get a list of node names in your cluster, use:
+The servers in the cluster are referred to by their PGD cluster node names. To get a list of node names in your cluster, use:
 
 ```SQL
 select node_name from bdr.node;
 ```
 
 !!! Tip
-More details about [`bdr.node`](/pgd/latest/reference/catalogs-visible#bdrnode) table are available in the reference section.
+For more details about the `bdr.node` table, see the [PGD reference](/pgd/latest/reference/catalogs-visible#bdrnode).
```

Comment:
You have broken the link to a specific entry in the reference

Reply:
I'm sorry, I don't really understand how this breaks the link by changing the link text? Please explain as there might be a mechanism I'm not familiar with.

Reply:
The link is from bdr.node to the reference documentation for bdr.node

But moving it to PGD reference, the link is broken between the actual text and the content and that goes forward in time polluting search engines and their perception of the text and the link (and making it look like a spammy replication).

Reply:
```suggestion
See  the [`bdr.node`](/pgd/latest/reference/catalogs-visible#bdrnode) table for details.
```
How's this?